.circleci/config.yml

```diff
@@ -0,0 +1,254 @@
+      - image: mongo:3.4
+        command: [mongod, --nojournal, --noprealloc, --smallfiles, --replSet=rs0]
+
+    environment:
+      TEST_MODE: "true"
+      MONGO_URL: mongodb://localhost:27017/testwithoplog
+      MONGO_OPLOG_URL: mongodb://localhost:27017/local
+
+    steps:
+      - attach_workspace:
+          at: /tmp
+
+      - checkout
+
+      - run:
+          name: Install dependencies
+          command: |
+            sudo apt-key adv --keyserver hkp://keyserver.ubuntu.com:80 --recv 0C49F3730359A14518585931BC711F9BA15703C6
+            echo "deb [ arch=amd64 ] http://repo.mongodb.org/apt/ubuntu trusty/mongodb-org/3.4 multiverse" | sudo tee /etc/apt/sources.list.d/mongodb-org-3.4.list
+            sudo apt-get update
+            sudo apt-get install -y mongodb-org-shell
+
+      - run:
+          name: Configure Replica Set
+          command: |
+            mongo --eval 'rs.initiate({_id:"rs0", members: [{"_id":1, "host":"localhost:27017"}]})'
+            mongo --eval 'rs.status()'
+
+      - restore_cache:
+          keys:
+            - node-modules-cache-{{ checksum ".circleci/config.yml" }}-{{ checksum "package.json" }}
+
+      - run:
+          name: Update NPM
+          command: |
+            sudo npm install -g npm
+
+      # - run:
+      #     name: NPM install
+      #     command: |
+      #       npm install --only=dev
+
+      - run:
+          name: Run Tests
+          command: |
+            for i in $(seq 1 5); do npm test && s=0 && break || s=$? && sleep 1; done; (exit $s)
+
+  test-without-oplog:
+    <<: *defaults
+    docker:
+      - image: circleci/node:4.8-browsers
+      - image: circleci/mongo:3.4
+
+    environment:
+      TEST_MODE: "true"
+      MONGO_URL: mongodb://localhost:27017/testwithoplog
+
+    steps:
+      - attach_workspace:
+          at: /tmp
+
+      - checkout
+
+      - restore_cache:
+          keys:
+            - node-modules-cache-{{ checksum ".circleci/config.yml" }}-{{ checksum "package.json" }}
+
+      - run:
+          name: Update NPM
+          command: |
+            sudo npm install -g npm
+
+      # - run:
+      #     name: NPM install
+      #     command: |
+      #       npm install --only=dev
+
+      - run:
+          name: Run Tests
+          command: |
+            for i in $(seq 1 5); do npm test && s=0 && break || s=$? && sleep 1; done; (exit $s)
+
+
+  deploy:
+    <<: *defaults
+    docker:
+      - image: circleci/node:4.8
+    # machine:
+    #     enabled: true
+
+    steps:
+      - checkout
+
+      - run: |
+          if [[ $CIRCLE_PULL_REQUESTS ]]; then exit 0; fi;
+
+          source .circleci/setartname.sh
+          source .circleci/setdeploydir.sh
+          bash .circleci/setupsig.sh
+          bash .circleci/namefiles.sh
+          # echo ".circleci/sandstorm.sh"
+
+          aws s3 sync $ROCKET_DEPLOY_DIR/ s3://download.rocket.chat/
+
+          bash .circleci/docker.sh
+          bash .circleci/update-releases.sh
```

Comment:
This and corresponding file I assume we no longer need, as that endpoint no longer exists.

Reply:
oh, good to know :+1:

Reply:
Disregard removing.  Just updated the contents of the file :)